templates/AureliaSpa/webpack.config.js

```diff
@@ -0,0 +1,74 @@
+var isDevBuild = process.argv.indexOf('--env.prod') < 0;
+var path = require('path');
+var webpack = require('webpack');
+var AureliaWebpackPlugin = require('aurelia-webpack-plugin');
+const srcDir = path.resolve('./ClientApp');
```

Comment:
Not sure why some of these are `const` and others are `var`. Ideally we'd support Node 0.x which is ES5. If we strictly need ES2015, can we be consistent about using `const`/`let`/etc?